### ISSUE TITLE
Add a warning/note in the setup that protein inputs may need to be equilibrated before running RBFE?

### ISSUE THREAD

**Issue by hannahbaumann** [OpenFreeEnergy/openfe#836]
Simulations can crash (give NaN errors) if the protein has not been equilibrated first. Maybe we could add a warning or note in the documentation that proteins may need to be equilibrated (short minimization and MD simulation) before setting up the free energy calculations

**Comment by IAlibay** [comment]
Sounds like a good idea to me!

**Comment by atravitz** [comment]
agreed! A system prep docs page of "this is what openfe assumes of your inputs" could be a helpful resource.
